test/opt/ccp_test.cpp

```diff
@@ -464,6 +464,60 @@ TEST_F(CCPTest, SSAWebCycles) {
 
   SinglePassRunAndMatch<opt::CCPPass>(spv_asm, true);
 }
+
+TEST_F(CCPTest, LoopInductionVariables) {
+  // Test reduced from https://github.com/KhronosGroup/SPIRV-Tools/issues/1143
+  // We are failing to properly consider the induction variable for this loop
+  // as Varying.
+  const std::string spv_asm = R"(
+               OpCapability Shader
+          %1 = OpExtInstImport "GLSL.std.450"
+               OpMemoryModel Logical GLSL450
+               OpEntryPoint Fragment %main "main"
+               OpExecutionMode %main OriginUpperLeft
+               OpSource GLSL 430
+               OpName %main "main"
+       %void = OpTypeVoid
+          %5 = OpTypeFunction %void
+        %int = OpTypeInt 32 1
+%_ptr_Function_int = OpTypePointer Function %int
+      %int_0 = OpConstant %int 0
+     %int_10 = OpConstant %int 10
+       %bool = OpTypeBool
+      %int_1 = OpConstant %int 1
+       %main = OpFunction %void None %5
+         %12 = OpLabel
+               OpBranch %13
+         %13 = OpLabel
+
+; This Phi should not have all constant arguments:
+; CHECK: OpPhi %int %int_0 {{%\d+}} {{%\d+}} {{%\d+}}
+         %22 = OpPhi %int %int_0 %12 %21 %15
```

Comment:
Should test like this:

```
; CHECK  [[phi:%\w+]] = OpPhi %int %int_0 {{%\w+}} [[induction_var:%\w+]] {{\d+}}
```
Then later as @alan-baker suggests, use [[phi]] in the OpSLessThan.
Also later add
```
; CHECK: [[induction_var]] = OpIAdd %int [[phi]] %int_1
```